randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -219,8 +228,8 @@ Extra - asset_id: collision_camera_004
       Trivial
   > Tunnel to Fan Room
       Any of the following:
-          Grapple Beam or Space Jump or Speed Booster or Infinite Bomb Jump (Beginner)
-          Spider Magnet and Use Spin Boost
+          Flash Shift or Speed Booster or Simple IBJ or Use Spin Boost
+          Grapple Beam and Grapple Movement (Beginner)
```

Comment:
Can also include magnet with grapple. If I understand correctly, with flash you have to wall jump then flash back into the wall to wall jump again to be able to get up? Something like that would be a beginner wall jump